chain_name: rarm
chain_root: CameraBottom_frame
chain_tip: r_wrist
marker_type: checkerboard
